.circleci/config.yml

```diff
@@ -324,7 +324,7 @@ jobs:
 
   deploy-locksmith-beanstalk:
     docker:
-      - image: circleci/python:3.6.4
+      - image: circleci/python:3.8.0b1
```

Comment:
I'm sort of concerned that it's upgrading us to a beta image

Reply:
yeah nah we shouldnt update this